cmd/rudder/rudder.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main // import "k8s.io/helm/cmd/rudder"
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/grpclog"
+
+	"k8s.io/helm/pkg/kube"
+	"k8s.io/helm/pkg/proto/hapi/release"
+	"k8s.io/helm/pkg/rudder"
+)
+
+var kubeClient = kube.New(nil)
+
+func main() {
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", rudder.GrpcAddr))
+	if err != nil {
+		grpclog.Fatalf("failed to listen: %v", err)
+	}
+	grpcServer := grpc.NewServer()
+	release.RegisterReleaseModuleServiceServer(grpcServer, &ReleaseModuleServiceServer{})
+
+	grpclog.Print("Server starting")
+	grpcServer.Serve(lis)
+	grpclog.Print("Server started")
+}
+
+// ReleaseModuleServiceServer provides implementation for release.ReleaseModuleServiceServer
+type ReleaseModuleServiceServer struct{}
+
+// Version is not yet implemented
+func (r *ReleaseModuleServiceServer) Version(ctx context.Context, in *release.VersionReleaseRequest) (*release.VersionReleaseResponse, error) {
+	grpclog.Print("version")
+	return &release.VersionReleaseResponse{}, nil
+}
+
+// InstallRelease creates a release using kubeClient.Create
+func (r *ReleaseModuleServiceServer) InstallRelease(ctx context.Context, in *release.InstallReleaseRequest) (*release.InstallReleaseResponse, error) {
+	grpclog.Print("install")
+	b := bytes.NewBufferString(in.Release.Manifest)
+	err := kubeClient.Create(in.Release.Namespace, b, 500, false)
```

Comment:
This timeout should be derived from the request, I did not include passing parameteres from InstallReleaseRequest sent to tiller to rudder yet.